### DISCUSSION TITLE
[macOS] Installing font-fantasque-sans-mono-nerd-font

### DISCUSSION THREAD

**Discussion by osalbahr** [ryanoasis/nerd-fonts#1303] [Q&A] [answered]
I want to install the nerd font that Garuda Linux uses, [FantasqueSansMono Nerd Font Mono](https://forum.garudalinux.org/t/what-nerd-font-does-garuda-use/12743/5) because I liked how the terminal looks. I followed [Install Nerd Fonts via Homebrew [updated & fixed] ](https://gist.github.com/davidteren/898f2dcccd42d9f8680ec69a3a5d350e) and was able to `brew install font-fantasque-sans-mono-nerd-font`. However, the emojis still don't render on Terminal and iTerm2. What should I do?

Here are the repos I used, with some modifications:

https://gitlab.com/garuda-linux/themes-and-settings/settings/garuda-fish-config

https://gitlab.com/garuda-linux/themes-and-settings/settings/garuda-starship-prompt

```
…ebula in ~ via C v14.0.3-clang via  v20.0.1 via  v20.3.0 via  via  v3.11.4
 ╰─λ brew install font-fantasque-sans-mono-nerd-font
Warning: Cask 'font-fantasque-sans-mono-nerd-font' is already installed.

To re-install font-fantasque-sans-mono-nerd-font, run:
  brew reinstall --cask font-fantasque-sans-mono-nerd-font

…n ~ via C v14.0.3-clang via  v20.0.1 via  v20.3.0 via  via  v3.11.4 took 2s
 ╰─λ ls -l .config/fish/config.fish
lrwxr-xr-x 40 osos  3 Jun 20:44  .config/fish/config.fish -> ../../git/garuda-fish-config/config.fish

…~ via C v14.0.3-clang via  v20.0.1 via  v20.3.0 via  via  v3.11.4 took 14ms
 ╰─λ ls -l .config/starship.toml
lrwxr-xr-x 43 osos  3 Jun 20:48  .config/starship.toml -> ../git/garuda-starship-prompt/starship.toml

…~ via C v14.0.3-clang via  v20.0.1 via  v20.3.0 via  via  v3.11.4 took 14ms
 ╰─λ
```

<img width="682" alt="Screenshot 2023-06-17 at 6 18 03 PM" src="https://github.com/ryanoasis/nerd-fonts/assets/54853250/ede075c8-5cb4-4d6c-9644-895ffafed64d">

<img width="682" alt="Screenshot 2023-06-17 at 6 17 52 PM" src="https://github.com/ryanoasis/nerd-fonts/assets/54853250/286bf3d4-f356-4d53-a2e1-44358275531c">

**Comment by Finii** [comment]
Codepoints `E256` `E24F` `E608` `E606`
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/42065fb4-da1b-4d43-b9d5-e565b948f973)

They are in the font file.

Are you sure you choose the font in the iTerm settings?
It does not look like it.

Fantasque:
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/8ab05a69-2651-462f-b916-32e15a6884f3)

Upper image - numeral four (`4`) and capital letter F (`F`) and more all different
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/4b868012-4547-4efb-b187-c5a395b559b2)


Lower image - Zero (`0`) and small letter A (`a`) clearly different
![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/4f063287-9040-4e6a-a1fc-3ef75ed563fa)

**Reply by osalbahr** [reply]
> Are you sure you choose the font in the iTerm settings?

No, I didn’t change any settings yet. I didn’t know I needed to do that. I thought the nerd font was only for the emojis

**Reply by Finii** [reply, marked as answer]
For iTerm you can specify an additional font just for the symbols.

But in any case, you need to select the Nerd Font somewhere.

<img width="1055" alt="Screenshot 2023-06-19 at 10 59 36" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/432c2100-75e2-4f6b-9b7c-b2be52c2cc26">

**Reply by Finii** [reply]
Here Terminal's settings, where you can/have to select the font:
<img width="779" alt="Screenshot 2023-06-19 at 11 02 49" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/4cfade62-1165-40d1-9c09-6ff7031ad56a">

**Reply by osalbahr** [reply]
Thx!
